### PR TITLE
Filtering

### PR DESCRIPTION
**Pull request by cphillips17**
Added the filtering - editing the landing and adding an FAQ view (in progress)

### CODE REVIEW

File: src/views/Landing.js [hunk 1]
```diff
@@ -203,19 +242,114 @@ function Landing() {
...
   );
 }
 
+function FirstTime() {
+  let [hasVisited, setVisited] = useState(
+    localStorage.getItem('raiportal:visited')
+  );
+  let setLSHasVisited = () => {
+    localStorage.setItem('raiportal:visited', true);
+  };
```
Comment (sshh12) [inline src/views/Landing.js:265]: This should also include a call to `setVisited(true)` (need to update both localstorage as well as already loaded state)

File: src/views/Landing.js [hunk 2]
```diff
@@ -203,19 +242,114 @@ function Landing() {
...
+          setModalVisible(true);
+        }}
+      >
+        Explore
+      </Button>
+    </>
+  );
+
+  const openNotification = () => {
+    setVisited(true);
```
Comment (sshh12) [inline src/views/Landing.js:293]: Everywhere (except inside setLSHasVisited) should call setLSHasVisited

**Review by sshh12 — CHANGES_REQUESTED** [review]
Looks good, small tweaks.

**Comment by sshh12** [comment]
May help to component-ize some of the duplicated parts of landing page.

**Comment by cphillips17** [comment]
Made the edits - for openNotification, the modal did not open if setLSHasVisited is called, will debug

**Comment by sshh12** [comment]
Merging so this PR doesn't block.

**Pull request merged**